admin/developers_tool_kit.php

```diff
@@ -206,7 +207,7 @@ function zen_display_files($include_root = false, $filetypesincluded = 1) {
 
         // use to debug for UTF-8 NO BOM on files: test search on a, e, s change if below to true
         if (false && htmlspecialchars($line, ENT_QUOTES, CHARSET) == '') {
-          echo '<br>SOMETHING BROKE in: ' . $file . '<br>on: ' . $line_num . ' - ' . $line . '<br>';
+          $output = '<br>SOMETHING BROKE in: ' . $file . '<br>on: ' . $line_num . ' - ' . $line . '<br>';
```

Comment:
Shouldn't this be `$output .=` instead of `$output =`?

Reply:
yes done.